spec/system/dismiss_topics_spec.rb

```diff
@@ -16,8 +16,8 @@
 
     expect(topic_list).to have_topic(topic)
     find(".dismiss-read").click
-    expect(dismiss_new_modal.dismiss_topics_checkbox).to be_checked
-    expect(dismiss_new_modal.dismiss_posts_checkbox).to be_checked
-    expect(dismiss_new_modal.untrack_checkbox).not_to be_checked
+    expect(dismiss_new_modal).to have_dismiss_topics_checked
+    expect(dismiss_new_modal).to have_dismiss_posts_checked
+    expect(dismiss_new_modal).not_to have_untrack_checked
```

Comment:
Note that this is an anti-pattern because Capybara will end up waiting the full default wait time trying to find something that is not going to be present.